geo/src/main/java/org/odk/collect/geo/SelectionMapActivity.kt

```diff
@@ -0,0 +1,38 @@
+package org.odk.collect.geo
+
+import android.app.Activity
+import android.content.Intent
+import androidx.activity.result.contract.ActivityResultContract
+import org.odk.collect.strings.localization.LocalizedActivity
+
+abstract class SelectionMapActivity : LocalizedActivity() {
```

Comment:
I think this should eventually become non `abstract` and all the config needed to select an item from a list of things will be passed using an implementation of `SelectItemFromMap`. Not quite there yet though!